src/globalStyles/newDataTableStyles.js

```diff
@@ -52,11 +52,18 @@ const newDataTableStyles = {
       color: DARK_GREY,
     },
   },
+  editableCellUnfocused: {
+    textAlign: 'right',
```

Comment:
Doesn't this need to be handled by above styles for cellText alignment?

Reply:
ah yes of course it does :(